server/pkg/api/store_apps.go

```diff
@@ -4,29 +4,224 @@ import (
 	"context"
 
 	"github.com/kube-tarian/kad/server/pkg/pb/serverpb"
+	"github.com/kube-tarian/kad/server/pkg/types"
 )
 
 func (s *Server) AddStoreApp(ctx context.Context, request *serverpb.AddStoreAppRequest) (
 	*serverpb.AddStoreAppResponse, error) {
-	return &serverpb.AddStoreAppResponse{}, nil
+
+	if request.AppConfig.AppName == "" {
+		s.log.Errorf("failed to add app config to store, %v", "App name is missing")
+		return &serverpb.AddStoreAppResponse{
+			Status:        serverpb.StatusCode_INTERNRAL_ERROR,
+			StatusMessage: "failed add app config to store, app name is missing",
+		}, nil
+	} else if request.AppConfig.Version == "" {
+		s.log.Errorf("failed to add app config to store, %v", "App version is missing")
```

Comment:
check with one condition both appname and version are not nil.
send error message "appName or version is empty"
do the same for other methods